admin_manual/configuration/user/user_types.rst

```diff
@@ -0,0 +1,117 @@
+  - Reactivated when a share is received.
+  - Administrators will be able to automate user cleanup ("**disabled for x days**").
+
+- Can use all clients.
+- Fully auditable in the enterprise edition.
+- Can be promoted to group administrator or administrator, but will still have no personal space.
+- Apps are specified by the admin (whitelist).
+
+.. _shared_with_guests_filter_label:
+
+.. note:: 
+   **The Shared with Guests Filter**
+
+   This filter makes it easy for sharers to view and remove their shares with a guest, which also removes their responsibility for guests. When all of a guest's shares are removed, the guest is then disabled and can no longer login.
+
+Standard User
+-------------
+
+- Is a regular user (from LDAP, ownCloud user backend, or another backend).
+- Has personal space. Permissions are granted by the administrator.
+- Shared space: Permissions as granted by sharer.
+- Apps: All enabled, might be restricted by group membership.
+
+Federated User
+--------------
+
+- Is not an internal user.
+- Can trust :ref:`a federated system <faq_federated_system_label>`.
+- Has access to shared space through users on the considered ownCloud system.
+- Can share data with the considered system (accept-/rejectable).
+
+ownCloud Group Administrator
+----------------------------
+
+- Is a regular user, such as from LDAP, an ownCloud user backend, or another backend.
+- Can manage users in their groups, such as adding and removing them, and changing quota of users in the group.
+- Can add new users to their groups and can manage guests.
+- Can enable and disable users.
+- Can impersonate users in their groups.
+- Custom group creation may be restricted to group admins.
+
+ownCloud Administrator
+----------------------
+
+- Is a regular user (from LDAP, ownCloud user backend, or another backend).
+- Can configure ownCloud features via the UI, such as sharing settings, app-specific configurations, and external storages for users.
+- Can manage users, such as adding and removing, enabling and disabling, quota and group management.
+- Can restrict app usage to groups, where applicable.
+- Configurable access to log files.
+- Mounting of external shares and local shares (of external file systems) is disabled by default.
+
+System Administrator
+--------------------
+
+- Is not an ownCloud user.
+- Has access to ownCloud code (e.g., ``config.php`` and apps folders) and command-line tool (:doc:`occ <../server/occ_command>`).
+- Configures and maintains the ownCloud environment (*PHP*, *Webserver*, *DB*, *Storage*, *Redis*, *Firewall*, *Cron*, and *LDAP*, etc.).
+- Maintains ownCloud, such as updates, backups, and installs extensions.
+- Can manage users and groups, such as via :doc:`occ <../server/occ_command>`.
+- Has access to the master key when storage encryption is used.
+- **Storage admin:** Encryption at rest.
+- **DB admin:** Calendar/Contacts etc. DB entries not encrypted.
```

Comment:
This is actually a drawback as we currently don't have an ability to 'separate' the DB admin access (like with the storage admin). Either we are honest and leave it there or we remove it. Up to you!

Reply:
I'd suggest leave it, and build on it later.